### DISCUSSION TITLE
Failed to load script: /_next/static/chunks/pages/index.js

### DISCUSSION THREAD

**Discussion by JhonSoto99** [reflex-dev/reflex#2461] [Q&A]
Failed to load script: /_next/static/chunks/pages/index.js at `reflex run`, this error suddenly started to appear and the project does not load.

it seems that the index.js file does not exist, the _app.js file is there instead

**Comment by picklelo** [comment]
Can you try deleting the `.web` directory and running again? Did you get any errors on the compile?

**Reply by jhonsoto-aflore** [reply]
@picklelo I have already tested deleting the `.web` folder and doing `reflex init` again and the problem persists, the application runs normally, it does not give any error in console.

**Reply by JhonSoto99** [reply]
@picklelo I have already tested deleting the `.web` folder and doing `reflex init` again and the problem persists, the application runs normally, it does not give any error in console.
